webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -318,10 +322,10 @@ export default class SystemUsers extends React.PureComponent<Props, State> {
         return (
             <div className='system-users__filter-row'>
                 <div className='system-users__filter'>
-                    <LocalizedInput
+                    <input
                         id='searchUsers'
                         className='form-control filter-textbox'
-                        placeholder={{id: t('filtered_user_list.search'), defaultMessage: 'Search users'}}
+                        placeholder={this.props.intl.formatMessage({id: t('filtered_user_list.search'), defaultMessage: 'Search users'})}
```

Comment:
I dont think props has intl object because we didn't wrap this component with `injectIntl`